.github/workflows/rust.yml

```diff
@@ -47,7 +48,7 @@ jobs:
           echo "cache_loaded=false" >> $GITHUB_ENV
         fi
     - name: Git checkout (reattach HEAD to Head Ref)
-      run: git checkout "$(echo ${{ github.head_ref }} | sed -E 's|refs/[a-zA-Z]+/||')"
+      run: run: git checkout "$(echo $HEAD_REF | sed -E 's|refs/[a-zA-Z]+/||')"
```

Comment:
You seem to have `run:` twice on this line which seems like a bug.

Reply:
geezus. thanks.